.github/workflows/build.yml

```
@@ -31,23 +31,11 @@ jobs:
       - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

Please bump `actions/checkout` too